buildScripts/docker/docker-compose-oracle.yml

```diff
@@ -10,4 +10,4 @@ services:
       WEB_CONSOLE: "true"
       DBCA_TOTAL_MEMORY: 1024
       PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
-      USE_UTF8_IF_CHARSET_EMPTY: "true"
\ No newline at end of file
+      USE_UTF8_IF_CHARSET_EMPTY: "true"
```

Comment:
What do you think this means?

Reply:
Oh, ok it has a message now.

Reply:
This was by mistake from other unrelated changes I was working on. I removed it.